api/src/pcapi/core/offerers/factories.py

```diff
@@ -53,6 +54,7 @@ class Meta:
     )
     contact = factory.RelatedFactory("pcapi.core.offerers.factories.VenueContactFactory", factory_related_name="venue")
     bookingEmail = factory.Sequence("venue{}@example.net".format)
+    dms_token = factory.LazyAttribute(lambda o: None if o.isVirtual else api.generate_dms_token())
```

Comment:
(détail) puisque tous les lieux y compris virtuels) auront un token : 

```suggestion
    dms_token = factory.LazyFunction(api.generate_dms_token)
```